localization/english/decisions/ritual_decisions_l_english.yml

```diff
@@ -1,13 +1,13 @@
 ﻿l_english:
-  ascension_ritual_decision:0 "Ritual of Ascension"
-  ascension_ritual_decision_desc:0 "You have reached the pinnacle of your current Elven level and are ready to ascend!\n\nRetreat to your chambers and begin your meditations. If the Divine Spark is receptive you will emerge as a higher being"
-  ascension_ritual_threshold_1_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! Your have achieved the absolute pinnacle form of the lower Elves and are ready to ascend to become something more.\n\n Upon completing this ascension you will enter the ranks of the Regal Fey, Elves that stand above the common folk, their mandate to rule self-evident"
-  ascension_ritual_decision_confirm:0 "I am ready"
+    ascension_ritual_decision:0 "Ritual of Ascension"
+    ascension_ritual_decision_desc:0 "You have reached the pinnacle of your current Elven level and are ready to ascend!\n\nRetreat to your chambers and begin your meditations. If the Divine Spark is receptive, you will emerge as a higher being."
+    ascension_ritual_threshold_1_decision_desc:0 "Behold your splendor! You have reached a Major Threshold! Your have achieved the absolute pinnacle form of the lower Elves and are ready to ascend to become something more.\n\n Upon completing this ascension you will enter the ranks of the Regal Fey, Elves that stand above the common folk, their mandate to rule blatantly self-evident."
```

Comment:
I dont know why but "blatantly" doesnt work for me here. completely subjective but it feels "crass" or "too strong".

I could go for "Overtly obvious" or "clear to all" or something else.

Reply:
I was writing it from the perspective of the regal fey; They seem a bit stuck up and "blatantly self-evident" fitted with the arrogant attitude. That's my interpretation though, not sure how you want to present them